freelist.go

```diff
@@ -49,7 +96,12 @@ func (f *freelist) count() int {
 
 // free_count returns count of free pages
 func (f *freelist) free_count() int {
-	return len(f.ids)
+	//use the forwardmap to get the total count
+	res := 0
```

Comment:
count := 0. do not use very generic variable naming like res.

Reply:
ok